README.md

```
@@ -1,7 +1,283 @@
+   ```bash
+   xcode-select --install
+   ```
+
+2. **Homebrew** (recommended for additional dependencies):
+
+   ```bash
+   /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+   ```
+
+3. **Rust Setup**:
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)